elements/rh-footer/rh-footer-lightdom.css

```diff
@@ -0,0 +1,35 @@
+rh-footer :is([slot^=links]) a {
+  gap: var(--rh-footer-links-gap, 10px);
+}
+
+rh-footer :is([slot^=links], [slot=footer-links-primary], [slot=footer-links-secondary]):is(h1, h2, h3, h4, h5) {
```

Comment:
@heyMP couldn't we do this in shadow DOM through the magic of `:is` e.g.

```css
::slotted(:is([slot^=links], [slot=footer-links-primary], [slot=footer-links-secondary]):is(h1, h2, h3, h4, h5)) {
  font-weight: 500;
  font-size: 14px;
  margin-top: 0;
  margin-bottom: 0;
}
```

or is this in lightdom for noscript?

Reply:
I went ahead and implemented this to show that it works, we'll revert if that's contrary to your original intention

Reply:
No, you're correct.  This should go in the shadow DOM. 👍